### PR TITLE
LP-2931 display current admin permissions

### PR DESCRIPTION
**Pull request by annas-usman**
[LP-2931](https://philanthropyu.atlassian.net/browse/LP-2931)

[Theme PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/605)

***

#### Description:
The permissions of the current admin have been moved out of the table.

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3261/workflows/workflows_id/jobs/3261/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   36 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3265/workflows/workflows_id/jobs/3265/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   37 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3267/workflows/workflows_id/jobs/3267/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   37 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by AliAdnanSohail — APPROVED** [review]

File: openedx/adg/admin_dashboard/admin_settings/helpers.py [hunk 1]
```diff
@@ -134,3 +139,37 @@ def get_title_for_action_completion(action_type):
...
+            span_start=HTML('<span class="admin-designation">'),
+            span_end=HTML('</span>'),
+        )
+    else:
+        business_unit = BusinessLine.business_lines_for_user(admin).first().title
+        permission = Text(_('{name}, {span_start}BU Admin of {business_unit}{span_end}')).format(
+            name=name,
+            span_start=HTML('<span class="admin-designation">'),
+            business_unit=business_unit,
+            span_end=HTML('</span>'),
```
Comment (javeriaejaz-arbisoft) [inline openedx/adg/admin_dashboard/admin_settings/helpers.py:169]: span_start & span_end should be define as constant.

File: openedx/adg/admin_dashboard/admin_settings/helpers.py [hunk 2]
```diff
@@ -134,3 +139,37 @@ def get_title_for_action_completion(action_type):
...
+    image = get_profile_image_urls_for_user(admin).get('medium')
+
+    if is_superuser_or_adg_admin(admin):
+        permission = Text(_('{name}, {span_start}ADG Admin{span_end}')).format(
+            name=name,
+            span_start=HTML('<span class="admin-designation">'),
+            span_end=HTML('</span>'),
+        )
+    else:
+        business_unit = BusinessLine.business_lines_for_user(admin).first().title
```
Comment (aightmunam) [inline openedx/adg/admin_dashboard/admin_settings/helpers.py:164]: Perhaps we should refactor this `business_lines_for_user` function to only return one business line?

**Review by javeriaejaz-arbisoft — APPROVED** [review]
LGTM 👏🏼  Added only a suggestion.

**Review by aightmunam — APPROVED** [review]
LGTM, just a small suggestion!

File: openedx/adg/admin_dashboard/admin_settings/helpers.py (hunk 2 above)
Comment (annas-usman) [reply openedx/adg/admin_dashboard/admin_settings/helpers.py:164]: @aightmunam actually we can add multiple business lines for an admin so this method would return all those.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3268/workflows/workflows_id/jobs/3268/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/constants.py (100%)
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   39 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/admin_dashboard/admin_settings/helpers.py (hunk 2 above)
Comment (aightmunam) [reply openedx/adg/admin_dashboard/admin_settings/helpers.py:166]: Didn't Andrew say that that would never happen? @annas-usman

**Review by aightmunam — DISMISSED** [review]

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3269/workflows/workflows_id/jobs/3269/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/constants.py (100%)
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   39 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by AliAdnanSohail — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3270/workflows/workflows_id/jobs/3270/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/constants.py (100%)
openedx/adg/admin_dashboard/admin_settings/helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
openedx/adg/admin_dashboard/admin_settings/views.py (100%)
-------------
Total:   39 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by aightmunam — APPROVED** [review]

**Pull request merged**